lib/caveatRegistry.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2018 Digital Bazaar, Inc.
+ *
+ * @author Christopher Lemmer Webber
+ * @author Ganesh Annan <gannan@digitalbazaar.com>
+ *
+ */
+'use strict';
+
+const utils = require('./utils');
+const vocab = require('./vocab');
+
+const {ExpireAtUri, RestrictActionUri} = vocab;
+
+const defaultCaveatRegistry = {};
+module.exports = defaultCaveatRegistry;
+
+defaultCaveatRegistry[ExpireAtUri] = checkExpireAtCaveat;
+defaultCaveatRegistry[RestrictActionUri] = checkRestrictActionCaveat;
+
+async function checkExpireAtCaveat({caveat, capability, proofPurposeOptions}) {
```

Comment:
I feel like these handlers need to return a meaningful error on failure.  Otherwise, how are errors going to be computed/returned at the higher level?